docs/reference/commandline/stats.md

```diff
@@ -85,6 +85,13 @@ b95a83497c91        awesome_brattain    0.28%               5.629MiB / 1.952GiB
 67b2525d8ad1        foobar              0.00%               1.727MiB / 1.952GiB   0.09%               2.48kB / 0B         4.11MB / 0B         2
 ```
 
+Running `docker stats` on container with name nginx and getting output in ``json`` format.
```

Comment:
I see there's double back-ticks around `json`. GitHub seems to be fine with that, but let me quickly fix that up and push to this branch (in case `docs.docker.com` doesn't like it when generating the HTML) 😅